### ISSUE TITLE
Format of email address in Basic Config is invalid

### ISSUE THREAD

**Issue by yolabingo** [dotCMS/core#21644]
**Describe the bug**
"Recover Password" email fails to send due to format of "Basic Config" -> "Email Address" -> 
`dotCMS Website <website@dotcms.com>` 

**To Reproduce**
1. Request a password reset email via Login form -> Recover Password
2. Email is not sent, dotCMS logs:

```
17:49:41.685  ERROR util.Mailer - javax.mail.internet.AddressException: Missing '<' in string ``"dotcms.org" <dotCMS Website <website@dotcms.com>>'' at position 49
javax.mail.internet.AddressException: Missing '<'
	at javax.mail.internet.InternetAddress.parse(InternetAddress.java:837) ~[mail.jar:1.5.2]
	at javax.mail.internet.InternetAddress.parseHeader(InternetAddress.java:663) ~[mail.jar:1.5.2]
	at javax.mail.internet.MimeMessage.getAddressHeader(MimeMessage.java:733) ~[mail.jar:1.5.2]
	at javax.mail.internet.MimeMessage.getFrom(MimeMessage.java:372) ~[mail.jar:1.5.2]
	at com.sun.mail.smtp.SMTPTransport.mailFrom(SMTPTransport.java:1630) ~[mail.jar:1.5.2]
	at com.sun.mail.smtp.SMTPTransport.sendMessage(SMTPTransport.java:1194) ~[mail.jar:1.5.2]
	at com.dotmarketing.util.Mailer.sendMessage(Mailer.java:363) ~[dotcms_22.02_999999.jar:?]
	at com.dotmarketing.util.EmailUtils.sendMail(EmailUtils.java:438) ~[dotcms_22.02_999999.jar:?]
	at com.dotcms.util.MessageAPI.sendMail(MessageAPI.java:28) ~[dotcms_22.02_999999.jar:?]
	...
```

**Comment by nollymar** [comment]
**Internal QA:** Passed

**Comment by bryanboza** [comment]
Fixed, tested on release-22.02 // Docker // FF
